nginx.template

```diff
@@ -9,10 +9,33 @@ events {}
 
 http {
     proxy_cache_path /tmp/nginx_cache levels=1:2 keys_zone=image_cache:10M max_size=100M inactive=40d;
+    proxy_cache_path /tmp/nginx_resp_cache levels=1:2 keys_zone=api_response_cache:2048M max_size=5G;
```

Comment:
Sikkert ikke noe problem, men er det 5G ledig på api-gateway serverene?

Reply:
Det er totalt 8GB, men en del blir jo brukt av diverse. Må ha sett feil for nå er det bare 4.8 ledig i test. Setter den ned til 4. I test er det heller ikke 2GB ledig minne, men tenker kanksje dette må tilpasses prodmiljøer.

Jeg skrur opp max_size for `image_cache`